comics/aggregator/crawler.py

```diff
@@ -298,3 +298,23 @@ def crawl_helper(self, feature_id, pub_date):
                 page = self.parse_page(release['url'])
                 url = page.src('img[itemprop="image"]')
                 return CrawlerImage(url)
+
+
+class NettserierCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at nettserier.no"""
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        page_url = 'https://nettserier.no/%s/' % (short_name)
```

Comment:
Just FYI: The parens around `short_name` here has no effect. To create a one-tuple, you must have a trailing comma, e.g. `(short_name,)`. The `%` formatting operator accepts both single values and tuples, so this works anyway.